Makefile

```diff
@@ -64,5 +61,5 @@ stop-local:
 
 .PHONY: compile-v3-contracts
 compile-v3-contracts:
-	cd consensus/polybft/polybftcontracts && npm install && npm run compile
+	cd v3-contracts && npm install && npm run compile && cd ..
```

Comment:
Rename to `core-contracts`?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/796/commits/54880bbd64b294f330dcbfbb334a64c1b54a4245